src/resource-descriptions/people.js

```diff
@@ -2,5 +2,40 @@ module.exports = {
   urlTemplates: {
     "self": "http://127.0.0.1:3000/people/{id}",
     "relationship": "http://127.0.0.1:3000/people/{ownerId}/relationships/{path}"
+  },
+
+  /**
+   * An optional function called on each resource provided by the client (i.e.
+   * in a POST or PATCH request) before it's sent to the adapter for saving. You
+   * can transform the data here as necessary or pre-emptively reject the
+   * request.
+   *
+   * @param {Resource} resource - The resource to be transformed
+   * @param {Object} req - The framework's request object
+   * @param {Object} res - The framework's response object
+   * #param {Funcation} superFn - The Super function
+   * @returns {Resource|undefined} - The transformed resource or undefined to
+   *   remove the resource from the response entirely
+   */
```

Comment:
I'd also add line like: `@throws Can throw an error (APIError or Error) to abort the request` tag in this docblock too.